plugins/woocommerce-admin/client/products/fills/pricing-section/pricing-field-taxes-class.tsx

```diff
@@ -44,44 +40,44 @@ export const PricingTaxesClassField = () => {
 	delete taxClassProps.checked;
 	delete taxClassProps.value;
 
+	if ( isTaxClassesResolving || taxClasses.length <= 0 ) {
```

Comment:
In response to your earlier comment here @louwie17 , it actually did need the fragment written since it could have returned `undefined`, unless I replaced it with a ternary returning `null`. I did a quick refactor to make this clearer.